### DISCUSSION TITLE
Managing accounts to get github.com and GHES, why?

### DISCUSSION THREAD

**Discussion by loewenstein** [community/community#30220] [Mobile] [answered]
I'd really like to just get my Github notifications from all possible sources.

Could this be done?
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.76.0 (75005810)
Device: iPhone 13 mini (iOS 15.6)
</details>

**Comment by taki-on** [comment, marked as answer]
The app can access a single account as a source for the data presented on the screen, and we provide a mechanism to switch accounts. We need to carefully consider potential security issues but it is true that it would be productive if users can scroll through all notifications from all possible sources. We will discuss if there is a safe way to realize it. Thank you for your feedback!

**Reply by loewenstein** [reply]
FWIW I just found out that long pressing the profile icon give quick access to account switching. It would still be nice to get an option to see all, just like mail apps allow to show a inbox from all accounts, but it's less urgent now from my POV.
